<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial_Set</class>
 <widget class="QMainWindow" name="Serial_Set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>359</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font：13pt;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_OpenButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:12pt;</string>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="m_StopBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_DataBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_CheckBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_BaudBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="m_SerNameBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>441</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
